docs/guides/execution-modes.mdx

```diff
@@ -7,11 +7,11 @@ description: An overview of the available execution modes in Qiskit Runtime; ses
 
 There are several ways to run workloads, depending on your needs.
 
-**Job mode**: A single primitive request of the estimator or the sampler made without a context manager. Circuits and inputs are packaged as primitive unified blocs (PUBs) and submitted as an execution task on the quantum computer.
+**Job mode**: A single primitive request of the estimator or the sampler made without a context manager. Circuits and inputs are packaged as primitive unified blocs (PUBs) and submitted as an execution task on the quantum computer. To run in job mode, specify `mode=backend` or pass the backend name when instantiating a primitive. See [Primitives examples](primitives-examples) for examples.
```

Comment:
We actually also deprecated passing the backend name when initializing a primitive. If you do `SamplerV2(mode="ibm_osaka")`, for example, it actually uses the default account under the cover, which could be unexpected to users. 

```suggestion
**Job mode**: A single primitive request of the estimator or the sampler made without a context manager. Circuits and inputs are packaged as primitive unified blocs (PUBs) and submitted as an execution task on the quantum computer. To run in job mode, specify `mode=backend` when instantiating a primitive. See [Primitives examples](primitives-examples) for examples.
```